libtransmission/announcer.c

```diff
@@ -1303,6 +1356,35 @@ static void tierAnnounce(tr_announcer* announcer, tr_tier* tier)
 ****
 ***/
 
+static bool multiscrape_too_big(char const* errmsg)
+{
+    unsigned int i;
```

Comment:
Could be declared inside the loop initializer (C99!). Should also be `size_t` since this is what `TR_N_ELEMENTS` returns.

Reply:
Good catch. Fixed dcd86bb4a3e57a82ad8f22b1386dde9016153f4e